api/models/reviewDao.js

```diff
@@ -0,0 +1,30 @@
+const appDataSource = require('./dataSource')
+
+const createUserReview = async(data, userId) => {
+
+    const createUserReview = await appDataSource.query(
+        `INSERT INTO
+            evaluation (
+                user_id,
+                store_id,
+                rating,
+                taste,
+                price,
+                service,
+                image,
+                content
+            ) VALUES (
+                ?,
+                ?,
+                ?,
+                ?,
+                ?,
+                ?,
+                ?,
+                ?
+            )
+            `, [userId, data.storeId, data.rating, data.taste, data.price, data.service, data.image, data.content])
+    return createUserReview
```

Comment:
```suggestion
    return appDataSource.query(
        `INSERT INTO
            evaluation (
                user_id,
                store_id,
                rating,
                taste,
                price,
                service,
                image,
                content
            ) VALUES (
                ?,
                ?,
                ?,
                ?,
                ?,
                ?,
                ?,
                ?
            )
            `, [userId, data.storeId, data.rating, data.taste, data.price, data.service, data.image, data.content])
```
마찬가지로 불필요한 변수 선언입니다. 그리고 현재 유저가 리뷰를 등록할 때 첨부하는 image는 database에 이미지를 그대로 저장하는 건가요?

Reply:
넵! 이미지 파일을 넣는 것으로 생각했는데, 용량 문제 또는 불러오기 문제에 있어서 파일 형식이 변환되거나 하면 좋을 것 같습니다. 관련하여 피드백 부탁 드리겠습니다!